Sofa/GL/Component/Rendering2D/src/sofa/gl/component/rendering2d/OglLabel.h

```diff
@@ -49,12 +49,11 @@ class SOFA_GL_COMPONENT_RENDERING2D_API OglLabel : public core::visual::VisualMo
     Data<sofa::type::RGBAColor> d_color; ///< The color of the text to display. (default='gray')
     Data<bool>                   d_selectContrastingColor ; ///< Overide the color value but one that contrast with the background color
     Data<unsigned int>           d_updateLabelEveryNbSteps; ///< Update the display of the label every nb of time steps
-    Data<bool>                   d_visible; ///< Is label displayed
```

Comment:
DeprecatedAndRemoved ?

Reply:
If we move forward for https://github.com/sofa-framework/sofa/pull/3934 I will use it :)

Reply:
I am confused. What do you plan to do with this Data? Currently, you have removed the class member and added an alias to `d_draw`. But, according to https://www.sofa-framework.org/community/doc/contributing-to-sofa/lifecycle-mechanisms/, when deleting a Data, its type must be `DeprecatedAndRemoved` to highlight the fact that a variable with this name existed, but has been removed.

Reply:
I plan to follow the lifecycle, so before deleting something, first deprecate it.

Reply:
I plan to follow the lifecycle, so before deleting something, first deprecate it.